portal-web/test/functional/com/liferay/portalweb/tests/mytests/MyDMTests.testcase

```diff
@@ -0,0 +1,56 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		Navigator.openURL();
+
+		task ("Given the DM Portlet is added to the page") {
+			JSONLayout.addPublicLayout(
+				groupName = "Guest",
+				layoutName = "DM Page");
+
+			JSONLayout.addWidgetToPublicLayout(
+				column = 1,
+				groupName = "Guest",
+				layoutName = "DM Page",
+				widgetName = "Documents and Media");
+
+			Navigator.gotoPage(pageName = "DM Page");
+
+			DMDocument.enableActionsMenuOnPortlet();
+
+			LexiconEntry.changeDisplayStyle(displayStyle = "list");
+		}
+	}
+
+	tearDown {
+		var testPortalInstance = PropsUtil.get("test.portal.instance");
+
+		if (${testPortalInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
```

Comment:
Hey, I think your teardown is missing to delete the document and media. You can revisit the documents and media files (.testcase) and take a look at the teardowns.

Reply:
Understood. Thank you for reviewing!